frontend/src/main/scala/bloop/bsp/BloopBspServices.scala

```diff
@@ -280,7 +280,8 @@ final class BloopBspServices(
               resourcesProvider = Some(true),
               buildTargetChangedProvider = Some(false),
               jvmTestEnvironmentProvider = Some(true),
-              jvmRunEnvironmentProvider = Some(true)
+              jvmRunEnvironmentProvider = Some(true),
+              canReload = Some(false)
```

Comment:
@jvican not sure about right thing to do here - `canReload = Some(false)` or `canReload = None`?

Reply:
I think `false` is correct here, since that issue was introduced for sbt and is not used in Bloop